tests/unit/test_utils.py

```diff
@@ -144,6 +145,40 @@ def test_clonable_git_url():
         == 'git://example.com'
 
 
+def test_inject_auth_git_url(monkeypatch) -> None:
```

Comment:
@lukaszachy ah missed this on first  review, could we add a test for this exception please?

```
                raise GitUrlError(f'Missing variable with credentials for "{url}"')
```

Reply:
fixed in 93a1973 and improved in d7b903f

Now it should state which variable is missing, eg.
`Missing "TMT_GIT_CREDENTIALS_VALUE_gl" variable with credentials for "https://gitlab.com/....."